languages/elixir/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have four tasks, all related to the time spent cooking the lasasgna.
+
+### 1. Define the expected oven time in minutes
+
+Define the `Lasagna.expected_minutes_in_oven/0` method that does not take any parameters and returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
+
+```elixir
+Lasagna.expected_minutes_in_oven()
+# => 40
+```
+
+### 2. Calculate the remaining oven time in minutes
+
+Define the `Lasagna.remaining_minutes_in_oven/1` method that takes the actual minutes the lasagna has been in the oven as a parameter and returns how many minutes the lasagna still has to remain in the oven, based on the expected oven time in minutes from the previous task.
+
+```elixir
+Lasagna.remaining_minutes_in_oven(30);
```

Comment:
I think the `;` is a remnant of the C# port, right?

```suggestion
Lasagna.remaining_minutes_in_oven(30)
```

Reply:
Yes it is a remnant, will fix